lib/beaker/host/unix.rb

```diff
@@ -26,7 +26,7 @@ def self.pe_defaults
         'puppetbin'        => '/opt/puppet/bin/puppet',
         'puppetbindir'     => '/opt/puppet/bin',
         'puppetsbindir'    => '/opt/puppet/sbin',
-        'systembindir'     => '/opt/puppet/bin',
+        'privatebindir'    => '/opt/puppet/bin',
```

Comment:
@kevpl I just noticed the PE default is wrong, it needs to be the same as AIO (/opt/puppetlabs/puppet/bin). Not an issue currently, but something we'll need to come back to.